pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,256 @@
+
+Oh, if you are still not sure about client or server side rendering, then you can read all about it here  [Website rendering for beginners](https://michals-corner.vercel.app/blog/website-rendering-for-beginners) (shameless plug ).
+
+I will cover the usage for:
+
+- [React](#react)
+- [Vue](#vue)
+- [Web Components](#wc)
+- [Standalone Bundle](#sb)
+
+The AsyncAPI sample file 👇 is used as JSON to make it easier to use on the JS side.
+All examples will use this same sample.
+
+```json
+{
+  "asyncapi": "2.4.0",
+  "info": {
+    "title": "Account Service",
+    "version": "1.0.0",
+    "description": "This service is in charge of processing user signups"
+  },
+  "channels": {
+    "user/signedup": {
+      "subscribe": {
+        "message": {
+          "$ref": "#/components/messages/UserSignedUp"
+        }
+      }
+    }
+  },
+  "components": {
+    "messages": {
+      "UserSignedUp": {
+        "payload": {
+          "type": "object",
+          "properties": {
+            "displayName": {
+              "type": "string",
+              "description": "Name of the user"
+            },
+            "email": {
+              "type": "string",
+              "format": "email",
+              "description": "Email of the user"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This is the expected look of the generated document 👇
+
+![document-template](/img/posts/event-driven-API-documentation-made-simple/document-template.webp "Document template")
+
+All usage examples from this article are available to check on the [asyncapi-docs-rendering-examples](https://github.com/m1ner79/asyncapi-docs-rendering-examples) repository.
+
+---
+
+### <a name="react"></a>React
+
+If you wish to render documentation from your AsyncAPI file in React application, then you need to use the [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react).
+
+1️⃣ To install the **React AsyncAPI component** run the command
+```bash
+npm install --save @asyncapi/react-component@next
+```
+
+2️⃣ Now, create a **index.js** file and type the following:
+
+```js
+import React from "react";
+import ReactDOM from "react-dom";
+
+import AsyncApiComponent from "@asyncapi/react-component";
+import "@asyncapi/react-component/styles/default.min.css";
+
+import { specMock } from "./testDoc";
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<AsyncApiComponent schema={specMock} />, rootElement);
+```
+
+Here we can see that the **React AsyncAPI component** is imported on **line 4**.
+
+If you are happy with AsyncAPI styling then you need to import their CSS pattern with:
+
+\```js
+import "@asyncapi/react-component/styles/default.min.css";
+\```
```

Comment:
Could you remove that `/` at the beginning of backticks? I added it to the suggestion, because markdown suggestions don't support three ``` and by this suggestion is broken.

Reply:
@magicmatatjahu that is fixed